assets/css/_comments.scss

```diff
@@ -50,20 +60,28 @@
 .avatar-rounded {
   @include size($rounded-avatar-size);
   border-radius: 50%;
-  margin-right: $tiny-spacing / 2;
+  margin-right: $tiny-spacing;
+  vertical-align: middle;
+}
+
+.avatar-rounded-large {
+  @include size($rounded-avatar-size-large);
+  margin-right: $small-spacing;
```

Comment:
Should this be handled by the new media object instead?

Reply:
Nice call!